app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  validates :author_id, presence: true
+  validates :title, presence: true
+  validates :text, presence: true
+
+  after_save :update_post_counter
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_post_counter
+    update(posts_counter: posts.count)
+  end
```

Comment:
- [ ] Kindly keep update_post_counter method should remain private. 🤓 as mentioned by the previous reviewer 💯🔥💪🏻 you can just add private keyword before it 🙏🏻

Reply:
done.